### PR TITLE
[CARBONDATA-4315] Supplementary information for add segment syntax .

### PR DESCRIPTION
**Pull request by bieremayi**
### Why is this PR needed?
 
Supplementary information for add segment syntax .
 
 ### What changes were proposed in this PR?

1. add segment option (partition)
2. segment-management-on-carbondata.md link addsegment-guide.md

 ### Does this PR introduce any user interface change?
 - No

 ### Is any new testcase added?
 - No

### CODE REVIEW

**Comment by CarbonDataQA2** [comment]
Can one of the admins verify this patch?

**Comment by chenliang613** [comment]
add to whitelist

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12602/job/ApacheCarbonPRBuilder2.3/6141/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 3.1, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_3.1/532/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_2.4.5/4398/

**Comment by Indhumathi27** [comment]
@bieremayi please raise a JIRA ticket and add the same to your PR

File: docs/segment-management-on-carbondata.md [hunk 1]
```diff
@@ -207,4 +208,4 @@ concept which helps to maintain consistency of data and easy transaction managem
     spark.sql("select count(empno) from carbon.input.segments.db.carbontable_Multi_Thread").show();
      }
    }
-  ```
+  ```
```
Comment (Indhumathi27) [inline docs/segment-management-on-carbondata.md:211]: please revert this change

File: docs/faq.md [hunk 2]
```diff
@@ -29,6 +29,7 @@
 * [Why different time zone result for select query output when query SDK writer output?](#why-different-time-zone-result-for-select-query-output-when-query-sdk-writer-output)
 * [How to check LRU cache memory footprint?](#how-to-check-lru-cache-memory-footprint)
 * [How to deal with the trailing task in query?](#How-to-deal-with-the-trailing-task-in-query)
+* [How to manage mix file format in carbondata table?](#How-to-manage-mix-file-format-in-carbondata-table)
```
Comment (Indhumathi27) [inline docs/faq.md:32]:
```suggestion
* [How to manage mixed file format in carbondata table?](#How-to-manage-mix-file-format-in-carbondata-table)
```

File: docs/faq.md [hunk 3]
```diff
@@ -213,6 +214,11 @@ spark.speculation is a group of configuration, that can monitor trailing tasks a
 
 spark.blacklist.enabled, avoid reduction of available executors due to blacklist mechanism.
 
+## How to manage mix file format in carbondata table
+
+[Heterogeneous format segments in carbondata](./addsegment-guide.md)
```
Comment (Indhumathi27) [inline docs/faq.md:219]:
```suggestion
Refer [Heterogeneous format segments in carbondata](./addsegment-guide.md)
```

File: docs/segment-management-on-carbondata.md (hunk 1 above)
Comment (bieremayi) [reply docs/segment-management-on-carbondata.md:211]: ok!

**Comment by bieremayi** [comment]
> 



> @bieremayi please raise a JIRA ticket and add the same to your PR

I've already done it.

**Comment by CarbonDataQA2** [comment]
Build Failed  with Spark 2.4.5, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_2.4.5/4399/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 3.1, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_3.1/533/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12602/job/ApacheCarbonPRBuilder2.3/6142/

File: docs/addsegment-guide.md [hunk 4]
```diff
@@ -27,16 +27,144 @@ Heterogeneous format segments aims to solve this problem by avoiding data conver
...
+	type INT                                    
+)                                              
+STORED AS parquet
+LOCATION 'hdfs://bieremayi/user/hive/warehouse/log_parquet_par';
+```
+
+Parquet File Location : 
+
+```
+25.1 K  75.2 K  /user/hive/warehouse/log_parquet_par/day=20211123/hour=12/type=0
```
Comment (chenliang613) [inline docs/addsegment-guide.md:98]: please remove these info : 25.1 K 75.2 K

File: docs/addsegment-guide.md [hunk 5]
```diff
@@ -27,16 +27,144 @@ Heterogeneous format segments aims to solve this problem by avoiding data conver
...
+8.7 K   26.2 K  /user/hive/warehouse/log_parquet_par/day=20211123/hour=12/type=1
```
Comment (chenliang613) [inline docs/addsegment-guide.md:99]: please remove : 8.7 K 26.2 K

**Comment by chenliang613** [comment]
@bieremayi  thanks for your pull request,  please revert some typo issues.

**Comment by CarbonDataQA2** [comment]
Build Failed  with Spark 2.4.5, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_2.4.5/4400/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 3.1, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_3.1/534/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12602/job/ApacheCarbonPRBuilder2.3/6143/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 3.1, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_3.1/535/

**Comment by CarbonDataQA2** [comment]
Build Failed  with Spark 2.3.4, Please check CI http://121.244.95.60:12602/job/ApacheCarbonPRBuilder2.3/6144/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_2.4.5/4401/

**Comment by brijoobopanna** [comment]
retest this please

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 3.1, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_3.1/554/

**Comment by CarbonDataQA2** [comment]
Build Failed  with Spark 2.3.4, Please check CI http://121.244.95.60:12602/job/ApacheCarbonPRBuilder2.3/6163/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_2.4.5/4420/

File: docs/addsegment-guide.md [hunk 6]
```diff
@@ -27,16 +27,144 @@ Heterogeneous format segments aims to solve this problem by avoiding data conver
...
+
+  - ##### PATH: 
+    User old table path.
+    
+    ``` 
+    OPTIONS('PATH'='hdfs://usr/oldtable')
+    ```
+
+  - ##### FORMAT:
+   User old table file format. eg : parquet, orc
```
Comment (Indhumathi27) [inline docs/addsegment-guide.md:57]: please format this line.
```suggestion
    User old table file format. eg : parquet, orc
```

File: docs/addsegment-guide.md [hunk 7]
```diff
@@ -27,16 +27,144 @@ Heterogeneous format segments aims to solve this problem by avoiding data conver
...
+
+    ```
+    OPTIONS('FORMAT'='parquet')
+    ```
+  - ##### PARTITION:
+   Partition info for partition table , should be form of "a:int, b:string"
```
Comment (Indhumathi27) [inline docs/addsegment-guide.md:63]:
```suggestion
    Partition info for partition table , should be form of "a:int, b:string"
```

**Comment by bieremayi** [comment]
retest this please

**Comment by CarbonDataQA2** [comment]
Build Failed  with Spark 3.1, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_3.1/559/

**Comment by CarbonDataQA2** [comment]
Build Failed  with Spark 2.3.4, Please check CI http://121.244.95.60:12602/job/ApacheCarbonPRBuilder2.3/6168/

**Comment by CarbonDataQA2** [comment]
Build Failed  with Spark 2.4.5, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_2.4.5/4425/

**Comment by Indhumathi27** [comment]
retest this please

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_2.4.5/4430/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12602/job/ApacheCarbonPRBuilder2.3/6174/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 3.1, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_3.1/565/

**Comment by chenliang613** [comment]
LGTM

**Comment by Indhumathi27** [comment]
LGTM
